tests/drone/getFilesChanged.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+CHANGED_UNIT_TESTS_ONLY=True
+CHANGED_DOCS_ONLY=True
+CHANGED_UNIT_TESTS_AND_DOCS_ONLY=True
+
+DIFFS=$(git diff origin/master --name-only)
+DOCS_ONLY_CHANGES=$(echo "$DIFFS" | grep -E '(docs/.*|changelog/.*)')
```

Comment:
I think this name is a bit confusing to me. This gives weather there is docs related changes (there can be other changes too), but the var name is `DOCS_ONLY_CHANGES` which (only part)  is not really true.

Reply:
only the changes in the files related to docs are extracted hence the name `docs_only_changes`. these changes have been used below to identify other conditions